src/Benefit.API/DTO/BeneficiaryResponse.cs

```diff
@@ -3,20 +3,37 @@
 namespace Benefit.API.DTO;
 public class BeneficiaryResponse
```

Comment:
Cara, mesmo as classes sendo estritamente relacionadas, não é melhor criar uma pasta "BeneficiaryResponse" e separar essas classes em arquivos dentro dela?
Porque seguindo essa lógica, o escopo fica muito amplo pra quem for codificar, pode induzir a um erro de padrão no projeto, pois os devs poderão fazer isso com qualquer classe que julguem estritamente relacionadas ou que só servem para aquele escopo.

Reply:
**Feito**